### DISCUSSION TITLE
Upgrage from 2.8 to 21.10 - Top menu deprecated Links

### DISCUSSION THREAD

**Discussion by mwielander** [centreon-archive/centreon-archived#10524] [Q&A] [answered]
Hello!

I have updated my Centreon Instance from 2.8 to 21.10.
Everything has worked so far.
Only the links in the top new menu do not point to the "new status page" as intended but still point to the old deprecated page.
How can I correct this behavior?

Greetings
Martin

![Screenshot_2021-12-29_18-08-27](https://user-images.githubusercontent.com/29795430/147686756-3ac458a7-1047-4f61-acdf-a505545bf965.jpg)

**Comment by ponchoh** [comment, marked as answer]
Hello 👋 

You should be able to switch this by unchecking the "Use deprecated pages" under "Administration  >  Parameters  >  My Account"

or adjust through the DB as needed:

```sql
MariaDB [centreon]> select contact_id,contact_alias,show_deprecated_pages from contact where show_deprecated_pages='1';
```

switch to 0
